DataFormats/Detectors/FIT/FDD/include/DataFormatsFDD/LookUpTable.h

```diff
@@ -214,21 +214,16 @@ class SingleLUT : public LUT
 
  public:
   static constexpr char sDetectorName[] = "FDD";
+  static constexpr char sDefaultCCDBpath[] = "http://ccdb-test.cern.ch:8080/";
+  static constexpr char sDefaultLUTpath[] = "FDD/LookUpTable";
+  inline static std::string sCurrentCCDBpath = sDefaultCCDBpath;
+  inline static std::string sCurrentLUTpath = sDefaultLUTpath;
```

Comment:
since your lookup table is constant within the run, I can understand why you make it singleton, hence static. 
But why do you need to have these paths as static? They are used only once at the Instance initialization, cannot you have theme simply as arguments of the Instance, defaulted to the values you need?

Reply:
I decided to keep Instance() without any arguments( `Instance(const std::string="default value" ...)` will call copy operation any time when LUT is needed ) and configuration via static fields in https://github.com/AliceO2Group/AliceO2/blob/4de04de161bd24ba8683c285f37bb4bc2d4917fd/Detectors/FIT/workflow/include/FITWorkflow/FITDataReaderDPLSpec.h#L52 looks reasonable. Should I change this?

Reply:
Why? If you define it as `Instance(const std::string& str = "default value" ...)` or `Instance(const std::string& str =  sDefaultCCDBpath)` then no copy will be involved, it will simply pass the reference on const.
I would prefer to minimize the number of statics. 
Anyway, if this PR is urgent, I can merge it is as it is, the cleanup can be done in other PR

Reply:
Oh, sure. Strange, I saw problems when I tried to use refs with defaults, thats why I decided to exclude default args... Could you merge this PR I will prepare cleanup in the next?